daily-programming/2024-05-07/main0507.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+using namespace std;
+int main(){
+  int n,k;
+  cin >> n >> k;
+  if(0<n,k && n,k<=100){

```

Comment:
こんな書き方できるの知らなかった笑
以下の書き方がメジャーだと思う。
```suggestion
if (0<n && n<=100 && 0<k && k<=100)
```